src/helpers/vote-results.ts

```diff
@@ -93,9 +74,8 @@ export function loadProposalAndSaveVoteResults(
       vote.nbYes = voteResults.value5;
       vote.nbNo = voteResults.value6;
 
-      vote.voteType = "OFFCHAIN";
-      // vote.proposal = maybeProposalId; //  proposal.id; //maybeProposalId; // proposalId.toHex();
-      // vote.offChain = proposal.id; //maybeProposalId;
+      vote.adapterName = "OffchainVotingContract";
```

Comment:
I'd just set the value using the variable you already have (`vote.adapterName = votingAdapterName;`) so there's no typo's.